thrimshim/thrimshim/main.py

```diff
@@ -381,6 +383,12 @@ def update_row(ident, editor=None):
 					for playlist in playlists
 				]
 				description_lines.append('') # blank line before footer
+			if new_row['thumbnail_mode'] == 'TEMPLATE':
+				template = database.query(conn, """
+					SELECT description FROM templates WHERE name = %s
+				""", new_row['thumbnail_template'])
+				if template.description:
```

Comment:
database.query returns a results object, not a single row.
```suggestion
				results = database.query(conn, """
					SELECT description FROM templates WHERE name = %s
				""", new_row['thumbnail_template'])
				template = results.fetchone()
				if template and template.description:
```